### PR TITLE
[Listing Service] CRUD Rooms

### PR DESCRIPTION
**Pull request by nvhai248**
- create room
- find one/find many room
- update room
- delete room

### CODE REVIEW

File: backend/nestjs/apps/listing-service/src/rooms/rooms.controller.ts
Comment (tuonglevan) [inline backend/nestjs/apps/listing-service/src/rooms/rooms.controller.ts:1]: This file should be stored in the `controllers` folder

File: backend/nestjs/apps/listing-service/src/rooms/rooms.service.spec.ts
Comment (tuonglevan) [inline backend/nestjs/apps/listing-service/src/rooms/rooms.service.spec.ts:1]: This file should be stored in the `services` folder

File: backend/nestjs/apps/listing-service/src/rooms/entities/room.entity.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  @Column()
+  geolocation: string;
+
+  @Column()
+  main_pic: string;
+
+  @IsUUID(4)
+  @IsNotEmpty()
+  @Column({ name: 'hotel_id' })
+  hotel_id: string;
```
Comment (panoti) [inline backend/nestjs/apps/listing-service/src/rooms/entities/room.entity.ts:23]: Can we use camel case to name field?

File: backend/nestjs/apps/listing-service/src/rooms/entities/room.entity.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  room_name: string;
+
+  @Column('text')
+  description: string;
+
+  @Column()
+  geolocation: string;
+
+  @Column()
+  main_pic: string;
```
Comment (panoti) [inline backend/nestjs/apps/listing-service/src/rooms/entities/room.entity.ts:18]: the same here

File: backend/nestjs/apps/listing-service/src/rooms/entities/room.entity.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { IsNotEmpty, IsUUID } from 'class-validator';
+import { AbstractEntity } from 'libs/database/src';
+import { Column, Entity, JoinColumn, ManyToOne } from 'typeorm';
+import { HotelEntity } from '../../hotels/entities/hotel.entity';
+
+@Entity({ name: 'rooms' })
+export class RoomEntity extends AbstractEntity<RoomEntity> {
+  @Column()
+  room_name: string;
```
Comment (panoti) [inline backend/nestjs/apps/listing-service/src/rooms/entities/room.entity.ts:9]: the same here

**Review by panoti — CHANGES_REQUESTED** [review]
Can we use camelCase when we name variable, field,... except global const variable?

File: backend/nestjs/apps/listing-service/src/rooms/entities/room.entity.ts (hunk 2 above)
Comment (tuonglevan) [reply backend/nestjs/apps/listing-service/src/rooms/entities/room.entity.ts:18]: @panoti I think property should be consistent the column field in the database.

**Pull request merged**